hack4impact-utk/compassion-ministries
# 13 create api route for creating an organization

# Description


## Relevant issue(s)




## Questions


## Type of change




# Checklist:




## Testing steps:
- Creating a new organization in the database.
- Run the server, and hit the endpoint at localhost:3000/api/organizations with {"name" : "nameSomething" }
- Verify that Organization was created
